code/modules/projectiles/ammunition/ballistic/shotgun.dm

```diff
@@ -147,7 +147,7 @@
 	return
 
 /obj/item/ammo_casing/shotgun/dart/bioterror
-	desc = "A shotgun dart filled with deadly toxins."
+	desc = "An improved shotgun dart filled with deadly toxins. Can be injected with up to 30 units of any chemical."
```

Comment:
changing the description does not change how they work.

Reply:
it inherits, the fix here is just the desc

```suggestion
	desc = "An improved shotgun dart filled with deadly toxins. Can be injected with up to 15 units of any chemical."
```